local/php/assert.go

```diff
@@ -0,0 +1,30 @@
+package php
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+)
+
+// assertIsPHPScript checks that the provided file is indeed a phar/PHP script (not a .bat file)
+func assertIsPHPScript(path string) bool {
```

Comment:
assert might seems a bit strong for developers coming from PHP and they might think this would stop the execution if the condition was not met.
WDYT renaming this function to `checkIsPHPScript` instead?

Reply:
Suits me fine!